test/integration/test_utils.mjs

```diff
@@ -435,6 +435,26 @@ async function getSerialized(page, filter = undefined) {
   return filter ? values.map(filter) : values;
 }
 
+async function getSerializedFromSerialized(page, serializedData) {
+  return page.evaluate(async data => {
+    const pageIndex = data.pageIndex;
+    const layer =
+      window.PDFViewerApplication.pdfViewer.getPageView(pageIndex)
+        .annotationEditorLayer.annotationEditorLayer;
+    // deserialize to get back the editor
+    const editor = await layer.deserialize(data);
+    layer.add(editor);
+    const value = editor.serialize();
+    // get serialized data from deserialized editor
+    for (const [k, v] of Object.entries(value)) {
+      if (ArrayBuffer.isView(v)) {
+        value[k] = Array.from(v);
+      }
+    }
+    return value;
```

Comment:
@Snuffleupagus You are right, I removed the layer.add(editor) line to avoid problems/surprises if this new helper function is used somewhere else. This function now only intent to check to serialize and deserialize with no side effect.